### ISSUE TITLE
Xcode 14 watchOS building for real devices(arm64) issues

### ISSUE THREAD

**Issue by jakov-clay** [VirgilSecurity/VirgilKeysiOS#58]
I'm having issues when building for watchOS in Xcode 14 for real devices (simulator is fine).  I get bunch of issues when linking VirgilCryptoFoundation for arm64 arch. In Xcode 13 works fine, probably related to bunch of changes Xcode 14 introduced for watchOS, now there is only 1 target instead of 2 when creating watchOS app, and new architecture.
<img width="807" alt="Screenshot 2022-10-06 at 12 29 21" src="https://user-images.githubusercontent.com/56873230/194290912-2999d5c2-a9d7-458f-92ff-07a91d1c16b8.png">

My Podfile:
```
use_frameworks!

target 'Demo' do
  platform :ios, '13.0'
  pod 'VirgilSDK', '8.1.0'
  
end

target 'DemoWatchOS WatchKit Extension' do
  platform :watchos, '7.0'
  pod 'VirgilSDK', '8.1.0'
  
end
```
Tested also with SPM, same result.

Maybe rebuilding libraries with Xcode 14 tools would help solve this issue?

**Comment by jakov-clay** [comment]
@SergeySeroshtan hey do you maybe have time to check this issue?

**Comment by SergeySeroshtan** [comment]
Will take a look

**Comment by jakov-clay** [comment]
@SergeySeroshtan were you able to reproduce issue? any ETA for this? My watch app is blocked by this so any info or help is helpful. Thanks!

**Comment by SergeySeroshtan** [comment]
Currently have issue with building low level libraries with updated environment. Going to resolve this ASAP. ETA until Oct 24.

**Comment by SergeySeroshtan** [comment]
Hi @jakov-clay what package manager do you use?

**Comment by SergeySeroshtan** [comment]
Met the same issue as - https://developer.apple.com/forums/thread/700583

**Comment by jakov-clay** [comment]
> Hi @jakov-clay what package manager do you use?

Cocoa Pods

**Comment by jakov-clay** [comment]
> Met the same issue as - https://developer.apple.com/forums/thread/700583

My issue occurs only in Xcode 14, everything seems to work normally in Xcode 13

**Comment by jakov-clay** [comment]
If I build for Any watchOS device I get a bit different error.

<img width="893" alt="Screenshot 2022-10-23 at 20 30 18" src="https://user-images.githubusercontent.com/56873230/197409422-d43d6d84-28e5-4779-b5f3-39324bd4424b.png">

I'm unsure why this happens. I can test on real watch device, if I can help let me know.

**Comment by SergeySeroshtan** [comment]
Going to accomplish this issue until tomorrow.

**Comment by jakov-clay** [comment]
@SergeySeroshtan how it's going? Any progress, problems, updates?

**Comment by SergeySeroshtan** [comment]
Hi @jakov-clay, I've got problems with Carthage package manager, that won't be able to fix quickly.
To unblock your's progress, I've released a new version for Cocoapods - 8.2.0 with arm64 arch for watchOS, please check it.

**Comment by jakov-clay** [comment]
@SergeySeroshtan I'm confirming that SDK works with latest Xcode on real devices! Thank you very much! I have some problems with my framework but Virgil is running normally.
